packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -83,49 +110,53 @@ export class TreeConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFie
 }
 
 /**
- * An editable view of a (version control style) branch of a shared tree.
+ * An editable view of a branch of a shared tree based on some schema.
+ *
+ * This schema--known as the view schema--may or may not align the stored schema of the document.
+ * Information about discrepancies between the two schemas is available via {@link TreeView.compatibility|compatibility}.
  *
- * This view is always in one of two states:
- * 1. In schema: the stored schema is compatible with the provided view schema. There is no error, and root can be used.
- * 2. Out of schema: the stored schema is incompatible with the provided view schema. There is an error, and root cannot be used.
+ * Application authors are encouraged to read [schema-evolution.md](../../docs/user-facing/schema-evolution.md) and
+ * choose a schema compatibility policy that aligns with their application's needs.
  *
  * @privateRemarks
- * From an API design perspective, `upgradeSchema` could be merged into `schematize` anb/or `schematize` could return errors explicitly.
+ * From an API design perspective, `upgradeSchema` could be merged into `viewWith` and/or `viewWith` could return errors explicitly on incompatible documents.
  * Such approaches would make it discoverable that out of schema handling may need to be done.
  * Doing that would however complicate trivial "hello world" style example slightly, as well as be a breaking API change.
  * It also seems more complex to handle invalidation with that pattern.
  * Thus this design was chosen at the risk of apps blindly accessing `root` then breaking unexpectedly when the document is incompatible.
- * If this does become a problem,
- * it could be mitigated by adding a `rootOrError` member and deprecating `root` to give users a warning if they might be missing the error checking.
  * @public
  */
 export interface TreeView<TSchema extends ImplicitFieldSchema> extends IDisposable {
 	/**
 	 * The current root of the tree.
 	 *
 	 * If in the out of schema state, accessing this will throw.
```

Comment:
I think this line needs updating. I think it should clarify this only works in the exactMatch case?